model/export/ColumnsProvider.php

```diff
@@ -92,15 +98,15 @@ public function getTestTakerColumns()
         $columns = [];
 
         // add custom properties, it contains the GROUP property as well
-        $customProps = $this->getClass(TAO_CLASS_SUBJECT)->getProperties();
+        $customProps = $this->getClass(TestTakerService::CLASS_URI_SUBJECT)->getProperties();
```

Comment:
it means that we need taoTestTaker in dependencies, also you need to be sure that we have needed dependencies for DeliveryAssemblyService and DeliveryContainerService

Reply:
@jbout it would be safe as taoTestTaker should became a dependency of taoOutcomeUi ?
The constant TAO_CLASS_SUBJECT was here before my change, but with my change we see this dependency.

The reason we have TAO_CLASS_SUBJECT here it's that in the export of results the test taker it's needed.

Reply:
taoTestTaker is already a dependency of taoOutcomeUi via taoDeliveryRdf->taoGroups

Reply:
@zagovorichev  can you please have another look at this PR ?